### DISCUSSION TITLE
Feature Request: Sort plugin store results by most relevant

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/console#65] [Plugin Store] [answered]
I regularly find the plugin store search results aren't as helpful as they could be because it's missing a sort by relevance (score). Could you consider this for a future update?

![Safari - 2023-02-14--09-29-23@2x](https://user-images.githubusercontent.com/101995330/218697015-48b8c424-32c0-4ec3-81ed-1138d9cd02b4.png)

**Comment by brianjhanson** [comment, marked as answer]
Hey @VeganSpeak, thanks for reaching out! We've just pushed an update to the plugin store that will sort search results by score from now on. 

![CleanShot 2024-06-17 at 12 32 02](https://github.com/craftcms/console/assets/1843073/7fb9a21d-9ea4-4f71-859f-421347981da7)
